course_gather/models.py

```diff
@@ -88,18 +126,55 @@ class MTUCourse(models.Model):
     '''
     The MTU Course
     '''
-    mtu_course_id = models.CharField(
+    mtu_course_crn = models.CharField(
                             max_length=50,
-                            help_text='The id of the MTU course')
-    mtu_course_name = models.CharField(
-                            max_length=50,
-                            help_text='The Name of the MTU course')
-    mtu_subject = models.CharField(
+                            help_text='The CRN of the MTU course')
+    mtu_course_subject = models.CharField(
                         max_length=50,
                         help_text='The subject of the MTU course')
-    mtu_credits = models.FloatField(
+    mtu_course_id = models.CharField(
+                        max_length=10,
+                        help_text='The course ID of the MTU course')
+    mtu_course_section = models.CharField(
+                            max_length=3,
+                            help_text='The section of the MTU course')
+    mtu_course_campus = models.CharField(
+                            max_length=2,
+                            default="1",
+                            help_text='The campus code of the MTU course')
+    mtu_course_credits = models.FloatField(
                         blank=True,
                         help_text='The amount of credits the MTU course is')
+    mtu_course_name = models.CharField(
+                            max_length=50,
+                            help_text='The Name of the MTU course')
+    mtu_course_days = models.CharField(
+                            max_length=7,
+                            help_text='The days the class is scheduled for')
+    mtu_course_time = models.CharField(
+                            max_length=40,
+                            help_text='The hours the MTU course runs')
+    mtu_course_capacity = models.IntegerField(
+                            help_text='The capacity of the class')
+    mtu_course_capacity_enrolled = models.IntegerField(
+                            help_text='The amount of students currently \
+                            enrolled in the course')
+    mtu_course_capacity_remaining = models.IntegerField(
+                            help_text='The seats remaining in the course')
+    mtu_course_instructor = models.CharField(
```

Comment:
Are we going to have a separate table for instructors, or is that out of the scope of this PR?

Reply:
Out of scope at the moment. There are some courses that are taught by upwards of 8 professors and have all of their names on it.

Reply:
Here's an example of such a course:
Instructor(s):
```

  | Jarvi , Cavaleri , Fiss , Flaspohler , Hyslop , Rudnicki , Schmierer
-- | --

```
And it is this class:
```

83435 | FW | 4800 | 0A | 1 | 2.000 | Commun. for Natural Resources
-- | -- | -- | -- | -- | -- | --



```